src/pretix/plugins/banktransfer/payment.py

```diff
@@ -530,6 +530,7 @@ def payment_pending_render(self, request: HttpRequest, payment: OrderPayment):
             'code': self._code(payment.order),
             'order': payment.order,
             'amount': payment.amount,
+            'info': payment.info_data,
```

Comment:
I would recommend a somewhat less ambiguous variable name than `info`. Either `payment_info`, `info_data` or `payment_info_data`.
```suggestion
            'payment_info': payment.info_data,
```